diff -ru graphite-web-0.9.9-upstream/setup.cfg graphite-web-0.9.9/setup.cfg
--- graphite-web-0.9.9-upstream/setup.cfg	2011-10-27 00:26:07.241584146 -0400
+++ graphite-web-0.9.9/setup.cfg	2011-10-27 00:26:52.903513723 -0400
@@ -1,3 +1,3 @@
 [install]
-prefix = /opt/graphite
-install-lib = %(prefix)s/webapp
+#prefix = /usr/share/graphite-web
+#install-lib = /usr/share/graphite-web/webapp
diff -ru graphite-web-0.9.9-upstream/webapp/graphite/settings.py graphite-web-0.9.9/webapp/graphite/settings.py
--- graphite-web-0.9.9-upstream/webapp/graphite/settings.py	2011-10-27 00:26:07.249549570 -0400
+++ graphite-web-0.9.9/webapp/graphite/settings.py	2011-10-27 00:29:43.025936926 -0400
@@ -21,19 +21,19 @@
 JAVASCRIPT_DEBUG = False
 
 # Filesystem layout (all directores should end in a /)
-WEB_DIR = dirname( abspath(__file__) ) + '/'
+WEB_DIR = '%PYTHON_SITELIB%/graphite/'
 WEBAPP_DIR = dirname( dirname(WEB_DIR) ) + '/'
 GRAPHITE_ROOT = dirname( dirname(WEBAPP_DIR) ) + '/'
-CONTENT_DIR = WEBAPP_DIR + 'content/'
+CONTENT_DIR = '/usr/share/graphite-web/webapp/content'
 CSS_DIR = CONTENT_DIR + 'css/'
 THIRDPARTY_DIR = WEB_DIR + 'thirdparty/'
 
-CONF_DIR = os.environ.get('GRAPHITE_CONF_DIR', GRAPHITE_ROOT + 'conf/')
-STORAGE_DIR = os.environ.get('GRAPHITE_STORAGE_DIR', GRAPHITE_ROOT + 'storage/')
+CONF_DIR = os.environ.get('GRAPHITE_CONF_DIR', '/etc/graphite/')
+STORAGE_DIR = os.environ.get('GRAPHITE_STORAGE_DIR', '/var/lib/graphite/storage/')
 LISTS_DIR = STORAGE_DIR + 'lists/'
 INDEX_FILE = STORAGE_DIR + 'index'
 WHITELIST_FILE = LISTS_DIR + 'whitelist'
-LOG_DIR = STORAGE_DIR + 'log/webapp/'
+LOG_DIR = '/var/log/graphite/webapp/'
 
 CLUSTER_SERVERS = []
 
Only in graphite-web-0.9.9/webapp/graphite: settings.py.orig
